### ISSUE TITLE
Detect ghostscript from TexLive 2019 and set corresponding environment

### ISSUE THREAD

**Issue by damca** [klatexformula/klatexformula#51]
On Windows 10, to get KLatexFormula working properly with Tex Live 2019's GhostScript I had to:

1. Change KLatex settings for gs to reference `C:\texlive\2019\tlpkg\tlgs\bin\gswin32c.exe` instead of `gswin32.exe`.

2. Create an environment `GS_LIB` with the value `C:\texlive\2019\tlpkg\tlgs\Resource\Init;C:\texlive\2019\tlpkg\tlgs\kanji;C:\texlive\2019\tlpkg\tlgs\lib;C:\texlive\2019\tlpkg\tlgs\bin;C:\texlive\2019\tlpkg\tlgs\fonts;C:\texlive\2019\tlpkg\tlgs\Resource;C:\texlive\2019\tlpkg\tlgs\iccprofiles;`

If these settings were included by default it may save other people a lot of time.

**Comment by phfaist** [comment]
Thanks for the heads up, I'll add that to the default search path. Can I ask you got the `GS_LIB` value? If there's an automatic way of detecting it it would be better than hard-coding it entirely (I imagine it could change for instance in future texlive versions).

**Comment by damca** [comment]
I don't have an automatic method unfortunately. But I can tell you how I found the necessary paths. 

My initial problem was the `GhostScript Can't find initialization file gs_init.ps` error. And I tried the solution [here](https://klatexformula.sourceforge.io/klfwiki/index.php/Old_Feedback_and_Issues), but it didn't work.

If I run the `gswin32c.exe` from the command prompt it also fails because it can't find the `gs_init.ps`. So maybe texlive's install of gs on Windows doesn't bind the necessary paths. I noticed that Ghostscript [install instructions on Windows](https://www.ghostscript.com/doc/current/Install.htm) requires access to the following:

```
gs#.##\bin
gs#.##\examples
gs#.##\lib
gs#.##\doc
gs#.##\Resource
fonts
```

Which were all inside the `C:\texlive\2019\tlpkg\tlgs` folder. So I added each of them (and extras) to the GS_LIB environment variable.

And I edited my original post, it had some extra folders in the GS_LIB that were not necessary. The current version has the minimal set needed for KLatex function.
